integration/traffic/interface.go

```diff
@@ -8,5 +8,5 @@ type HistoricalSpeedQuerier interface {
 	// QueryHistoricalSpeed return the speed for a way at a specified time
 	// wayID: positive means forward, negative means backward
 	// t: UTC time
```

Comment:
Nit: may be we could make what is `forward` more clear by following description:
```
wayID: positive means travel forward, which is travel following edge's point sequence, negative means backward
```

Reply:
Good suggestion, done!